src/vl-wizard-pane.js

```diff
@@ -245,25 +245,28 @@ export class VlWizardPane extends vlElement(HTMLElement) {
     const previousAction = this._previousAction;
     if (previousAction) {
       this._previousAction.addEventListener('click', () => {
-                this.isPreviousPaneDisabled ? this.disablePreviousPane() : this.enablePreviousPane();
+        this.isPreviousPaneDisabled ? this.disablePreviousPane() : this.enablePreviousPane();
       });
     }
     if (nextAction) {
       this._nextAction.addEventListener('click', () => {
-                this.isNextPaneDisabled ? this.disableNextPane() : this.enableNextPane();
+        this.isNextPaneDisabled ? this.disableNextPane() : this.enableNextPane();
       });
     }
   }
 
   _observeActiveClass(callback) {
     const observer = new MutationObserver((mutations) => {
-      const wasActive = (mutation) => this._isActive(mutation.oldValue ? mutation.oldValue.split(' ') : []);
-      const isActive = (mutation) => mutation.target.isActive;
-      if (mutations.some((mutation) => !wasActive(mutation) && isActive(mutation))) {
-        callback();
+      if (this.isActive) {
+        if (!this.__active && mutations.some((mutation) => mutation.target.isActive)) {
```

Comment:
if (this.isActive) en dan daarbinnen if (!this.active), dan zou kik denken, nee da moet altijd false zijn. dus ik zou proberen iets aan de naamgeving te doen.

Reply:
Is reeds opgemerkt en opgelost in PR #48. Ik was van die PR vertrokken, maar zal een merge uitvoeren.